packages/core/src/v3/runMetadata/metadataStream.ts

```
@@ -2,65 +2,48 @@ export type MetadataOptions<T> = {
   version?: "v1" | "v2";
 };
 
 export class MetadataStream<T> {
   private controller = new AbortController();
-  private serverQueue: Array<Promise<IteratorResult<T>>> = [];
-  private consumerQueue: Array<Promise<IteratorResult<T>>> = [];
-  private serverIterator: AsyncIterator<T>;
-  private consumerIterator: AsyncIterator<T>;
+  private serverStream: ReadableStream<T>;
+  private consumerStream: ReadableStream<T>;
   private streamPromise: Promise<void | Response>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)